packages/gatsby/src/components/hooks.tsx

```diff
@@ -0,0 +1,35 @@
+import {StaticQuery, graphql} from 'gatsby';
+import React                  from 'react';
+
+export function Hooks() {
+  return (
+    <StaticQuery
+      query={graphql`
+        query MyQuery {
+          yarnIntrospection(id: {eq: "0f47090c-afdf-5a02-98f8-047adfe6297f"}) {
+            value {
+              name
+              definition
+              comment
+              file
+            }
+          }
+        }
+      `}
+      render={data => <>
+        {data.yarnIntrospection.value.map(({name, comment, definition}) => (
+          <div key={name}>
+            <h3 id={`hook-${name}`}><code>{name}</code></h3>
```

Comment:
Is it intended that these `h3`s don't have clickable anchors? :thinking:

Reply:
The anchors were added by the Gatsby plugin, so couldn't be added without copy-paste. I added them back, although it makes the code a little uglier.